app/controllers/test_users_controller.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+require 'salesforce_api'
+
+class TestUsersController < ApplicationController
+  before_action :get_program_select, only: [:index]
+  before_action :set_env, only: [:index, :post, :get_program_tas]
+
+  layout 'admin'
+
+  def index
+    authorize :TestUser
+    @role_options = ["Fellow", "Leadership Coach", "Teaching Assistant", "Coach Partner", "Staff", "Faculty"]
+      .map { |role| [role, role.split(" ").join] }
```

Comment:
took me a second to understand what this was doing, i think a more idiomatic way would be with gsub:
```suggestion
      .map { |role| [role, role.gsub(" ", "")] }
```

Reply:
That does definitely seem better, but I think I am going to get rid of this all together based on [this comment ](https://github.com/bebraven/platform/pull/1081#discussion_r875148747 )